### DISCUSSION TITLE
How the relationship works in Budibase

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7487] [Help]
I want to get user information who has placed order using the Budibase Database.
How Can I get user's information from the orders table, I want to know more about relationship in the budibase

**Comment by R2bEEaton** [comment]
Hi @DevrayePatil,

A good place to start would be the documentation pages on [Relationships](https://docs.budibase.com/docs/relationships) and on [Created By / Updated By columns](https://docs.budibase.com/docs/users-table#created-by--updated-by-columns).

As for getting the "Created By" field from a row, this can be done easily with Handlebars. First, have a look at the entire row so you can get an idea of what the structure of a row looks like in Budibase. Add a new paragraph element under your data provider and enter the following, making sure to replace with the correct provider name.

```
{{ PROVIDER.Rows }}
```
This shows all of the rows.

Now, you can drill down into the "Created By" column (which is an automatically generated column by Budibase, unless you disabled it) using the following Handlebars. Remove the paragraph and add a Repeater within the Data Provider (for example). Now, within the repeater, add back a new paragraph.

```
{{ REPEATER.TABLENAME.Created By }}
```
This will show you all the data for the user that created the table. To get the name or email in a more useful format use:
```
{{ REPEATER.TABLENAME.Created By.0.primaryDisplay }}
```

Hope this helps! Reply with any questions or mark this answer using the check mark above to help future readers.

**Comment by DevrayePatil** [comment]
I want to get all the user's information present in the users table by this I am getting only primaryDisplay
I want to get user's hole Information how can I get this

**Reply by R2bEEaton** [reply]
Hi, for future reference, please use the reply feature below an answer instead of adding a new answer.

This can be done by nesting another data provider within your original one for the users table. You can then filter it by `_id` = `{{ REPEATER.TABLENAME.Created By.0._id }}` from my previous response. This way, you now have a data provider that returns all the information about the filtered user.
